.vscode/tasks.json

```diff
@@ -0,0 +1,29 @@
+{
+	"version": "2.0.0",
+	"tasks": [{
+		"label": "Start dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml up -d",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up -d"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		}
+	  }, {
+		"label": "Stop dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml down",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml down"
+		},
+		"group": "none",
+		"presentation": {
+		  "reveal": "always",
+		  "panel": "new"
+		}
+	  }
+	]
```

Comment:
Style: formatting seems off, can you have a pass with VSCode's JSON formatter?

Reply:
Yeah, I've got an issue with the formatter for json: when I set the default one, it has a different behavior depending on the file.

In the settings.json, I add 
```json
"[jsonc]": {
  "editor.defaultFormatter": "vscode.json-language-features"
}
```

When formatting settings.json, it uses 2 spaces, but for tasks.json it uses 1 tab

Reply:
I see, have you tried using Prettier? Though I was actually thinking of the way the various brackets open and close.
E.g. the array is opened as follows on line 3: 
```json
"tasks": [{
```
the items are separated on the same line at line 15:
```
  }, {
```
The indentation is hence shifted when closing (lines 26-28):
```
		}
	  }
	]
```

We may want to setup Prettier for JSON files as well. Since it is quite opinionated (and configurable), it would always ensure that e.g. arrays are formatted in the same way.

Reply:
For instance, Prettier formatted the current code like this, which looks more regular imo:

```jsonc
  [
    {
      "label": "Start dev container",
      "type": "shell",
      "command": "docker compose --file .devcontainer/docker-compose.yaml up",
      "windows": {
        "command": "docker compose --file .devcontainer\\docker-compose.yaml up"
      },
      "group": "none",
      "presentation": {
        "reveal": "always",
        "panel": "new"
      },
      "runOptions": {
        "runOn": "folderOpen"
      }
    },
    {
      "label": "Stop dev container",
      "type": "shell",
      "command": "docker compose --file .devcontainer/docker-compose.yaml down",
      "windows": {
        "command": "docker compose --file .devcontainer\\docker-compose.yaml down"
      },
      "group": "none",
      "presentation": {
        "reveal": "always",
        "panel": "new"
      }
    }
  ]
```

Reply:
Solved in [83614ae](https://github.com/clicepfl/clic-website-v2/pull/22/commits/83614aeaafc71de890e17a7c103c51715456aff9)